soccer/planning/SingleRobotPathPlanner.cpp

```diff
@@ -0,0 +1,36 @@
+#include "SingleRobotPathPlanner.hpp"
+#include "DirectTargetPathPlanner.hpp"
+#include "RRTPlanner.hpp"
+
+namespace Planning {
+
+REGISTER_CONFIGURABLE(SingleRobotPathPlanner);
+
+ConfigDouble* SingleRobotPathPlanner::_goalChangeThreshold;
+ConfigDouble* SingleRobotPathPlanner::_replanTimeout;
+
+void SingleRobotPathPlanner::createConfiguration(Configuration* cfg) {
+    _replanTimeout = new ConfigDouble(cfg, "PathPlanner/replanTimeout", 5);
+    _goalChangeThreshold =
+        new ConfigDouble(cfg, "PathPlanner/goalChangeThreshold", 0.025);
+}
+
+std::unique_ptr<Planning::SingleRobotPathPlanner> PlannerForCommandType(
+    Planning::MotionCommand::CommandType type) {
```

Comment:
Ahh gotcha.  Well this method is intended to make a new one each time - I think it should be the user of this function's responsibility to re-use the same planner over and over again as needed.  Right now IndependentMultiRobotPathPlanner only calls this when the command type changes.